contracts/teller-network/ArbitratorLicense.sol

```diff
@@ -0,0 +1,93 @@
+
+pragma solidity ^0.5.8;
+
+import "./License.sol";
+
+/**
+* @title ArbitratorLicense
+* @dev Contract for management of an arbitrator license
+*/
+contract ArbitratorLicense is License{
+
+    enum RequestStatus {AWAIT,ACCEPTED,REJECTED}
+
+    struct Requests{
+        address seller;
+        address arbitrator;
+        RequestStatus status;
+    }
+
+	struct ArbitratorLicenseDetails {
+        uint id; 
+        bool acceptAny; // accept any seller
+        address[] accepted; // addresses of accepted sellers
+    	bool isActive; // is license active?
+    }
+
+    mapping(address => ArbitratorLicenseDetails) arbitratorlicenseDetails;
+
+    Requests[] public requests;
+    mapping(uint => uint[]) public requestsById;
+
+    event ArbitratorRequested(uint id, address seller, address arbitrator);
+    event ArbitratorLicensed(uint id, bool acceptAny);
+    event RequestAccepted(address arbitrator, address seller);
+
+    /**
+     * @notice Buy an arbitrator license
+     * @param _acceptAny indicates does arbitrator allow to accept any seller/choose sellers 
+     */
+    function buyLicense(bool _acceptAny) public {
```

Comment:
if someone calls `buy()` instead of `buyLicense` I think it might create an arbitration license that has an invalid state due to not containing details.

What do you think of overriding `buy` in this contract so it does something like:

```
function buy() public {
   buyLicense(true);
}
```